src/Apps/Collect/CollectApp.tsx

```diff
@@ -77,6 +79,14 @@ export class CollectApp extends Component<CollectAppProps> {
   }
 }
 
+const h1Style = {
+  fontStyle: "inherit",
+  fontFamily: "inherit",
+  fontWeight: 400,
```

Comment:
fontWeight should be able to inherit from `Serif` as well.

Reply:
this is a weird one—when I set it to `"inherit"`, TS gives me this lovely message https://cl.ly/924cf404caf5

seems like it might be an issue with react and not with our code ¯\_(ツ)_/¯

Reply:
or rather with TS

Reply:
You could use styled components here rather than passing `style` attributes directly, this is usually the pattern taken in this repo:
```
const H1 = styled.h1`
  font-style: inherit;
  font-family: inherit;
...
`
```

Reply:
ah that makes sense! and actually, it looks like Chris has addressed this globally in [another PR](https://github.com/artsy/reaction/pull/1970/files#r252521612), so I should probably go ahead and remove these styles and keep this PR to just the capitalization change

Reply:
Great, glad to know this is now global!